backend/app/services/knowledge/knowledge_service.py

```
@@ -516,8 +546,16 @@ def delete_knowledge_base(
         if not kb:
             return False
 
-        # Only creator can delete knowledge base
-        if kb.user_id != user_id:
+        # Check permission for organization knowledge base (admin only)
+        if kb.namespace == "organization":
+            from app.models.user import User
+
+            user = db.query(User).filter(User.id == user_id).first()
+            if not user or user.role != "admin":
+                raise ValueError("Only admin can delete organization knowledge base")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)